src/content/app/help/Help.tsx

```diff
@@ -92,14 +123,52 @@ const AppBar = () => {
   );
 };
 
+const HelpHistoryButtons = () => {
+  const previousHistoryItem = useSelector(getPreviousHistoryItem);
+  const nextHistoryItem = useSelector(getNextHistoryItem);
+  const dispatch = useDispatch();
+
+  const onHistoryBack = () => {
+    if (previousHistoryItem) {
+      dispatch(moveBackInHistory());
+      dispatch(goBack());
+    }
+  };
+
+  const onHistoryForward = () => {
+    if (nextHistoryItem) {
+      dispatch(moveForwardInHistory());
+      dispatch(goForward());
+    }
+  };
+
+  return (
+    <HistoryButtons
+      onHistoryBack={onHistoryBack}
+      onHistoryForward={onHistoryForward}
+      hasPrevious={!!previousHistoryItem}
+      hasNext={!!nextHistoryItem}
+    />
+  );
+};
+
 const MainContent = (props: {
   article: ArticleData;
   breadcrumbs: string[];
 }) => {
   const { article, breadcrumbs } = props;
-  if (article.type !== 'article' && article.type !== 'video') {
+  const dispatch = useDispatch();
+
+  const isHelpContent = article.type === 'article' || article.type === 'video';
+
+  if (!isHelpContent) {
     return null;
   }
+
+  const onRelatedItemClick = (reference: RelatedArticleData) => {
+    dispatch(push(reference.url));
+  };
```

Comment:
This function no longer serves any purpose, right?

Reply:
It still does as we need to push the URL in the `RelatedArticles` component.

Reply:
Why would you need to explicitly push a url on click? Isn't it what the `useEffect` that we added is doing here?

https://github.com/Ensembl/ensembl-client/blob/26ebadbafc76d71c5aa26c2cfee8a7ba406fe0b8/src/content/app/help/Help.tsx#L81-L85

Reply:
It is adding that to the help navigation history and not the browser history. For `back` and `forward` navigation we use `goBack` and `goForward` from `connected-react-router`.

https://github.com/Ensembl/ensembl-client/blob/26ebadbafc76d71c5aa26c2cfee8a7ba406fe0b8/src/content/app/help/Help.tsx#L134

Reply:
Had a discussion with @azangru and seems like I misunderstood the issue here. I need to get rid of `onRelatedItemClick` as it is not needed anymore. The link will be pushed to the browser history in the `RelatedArticles` components.

Reply:
This is done now

✅